plugins/content/emailcloak/src/Extension/EmailCloak.php

```diff
@@ -38,8 +39,9 @@ final class EmailCloak extends CMSPlugin implements SubscriberInterface
     public static function getSubscribedEvents(): array
     {
         return [
-            'onContentPrepare' => 'onContentPrepare',
-            'onFinderResult'   => 'onFinderResult',
+            'onContentPrepare'                => 'onContentPrepare',
+            'onFinderResult'                  => 'onFinderResult',
+            'onCustomFieldsAfterPrepareField' => 'onCustomFieldsAfterPrepareField'
```

Comment:
```suggestion
            'onCustomFieldsAfterPrepareField' => 'onCustomFieldsAfterPrepareField',
```
Sorry, missed that with my last proposal. But this should finally fix PHPCS: